Patches/Red Horse Faction Elite Crew/RH_EliteCrew_CE_Patch_RangedIndustrial_Grenades.xml

```diff
@@ -0,0 +1,281 @@
+						<fragments>
+							<Fragment_Small>31</Fragment_Small>
+						</fragments>
+					</li>
+				</value>
+			</li>		
+
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+				<defName>RNThrown_RGDNade</defName>
+				<statBases>
+					<Mass>0.31</Mass>
+					<Bulk>0.93</Bulk>
+					<MarketValue>11.19</MarketValue>
+					<RangedWeapon_Cooldown>1</RangedWeapon_Cooldown>
+					<SightsEfficiency>0.45</SightsEfficiency>
+				</statBases>
+				<Properties>
+					<label>throw RGD-5 grenade</label>
+					<verbClass>CombatExtended.Verb_ShootCEOneUse</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<range>10.0</range>
+					<minRange>4</minRange>
+					<warmupTime>0.8</warmupTime>
+					<noiseRadius>4</noiseRadius>
+					<ai_IsBuildingDestroyer>true</ai_IsBuildingDestroyer>
+					<soundCast>ThrowGrenade</soundCast>
+					<targetParams>
+						<canTargetLocations>true</canTargetLocations>
+					</targetParams>
+					<defaultProjectile>RNProj_RGDGrenade</defaultProjectile>
+					<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
+					<rangedFireRulepack>Combat_RangedFire_Thrown</rangedFireRulepack>
+				</Properties>
+				<weaponTags>
+					<li>CE_AI_Grenade</li>
+					<li>CE_OneHandedWeapon</li>
+					<li>RN_RGDGrenade</li>
+				</weaponTags>
+			</li>
+
+			<!-- ========== Patch C4 (Elite) ========== -->
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RNProj_C4CSGO_Elite"]/graphicData/texPath</xpath>
+				<value>
+					<texPath>Things/Projectile/Proj_CSGOC4_Elite</texPath>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RNProj_C4CSGO_Elite"]/projectile</xpath>
+				<value>
+					<projectile Class="CombatExtended.ProjectilePropertiesCE">
+						<explosionRadius>4</explosionRadius>
+						<damageDef>Bomb</damageDef>
+						<damageAmountBase>328</damageAmountBase>
+						<explosionDelay>500</explosionDelay>
+						<dropsCasings>true</dropsCasings>
+						<casingMoteDefname>Mote_GrenadePin</casingMoteDefname>
+						<applyDamageToExplosionCellsNeighbors>true</applyDamageToExplosionCellsNeighbors>
+						<speed>2</speed>
+					</projectile>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RNProj_C4CSGO_Elite"]/comps</xpath>
+				<value>
+					<li Class="CombatExtended.CompProperties_ExplosiveCE">
+						<fragments>
+							<Fragment_GrenadeFrag>345</Fragment_GrenadeFrag>
+						</fragments>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RNThrown_C4CSGO_Elite"]</xpath>
+				<value>
+					<thingClass>CombatExtended.AmmoThing</thingClass>
+					<stackLimit>25</stackLimit>
+					<resourceReadoutPriority>First</resourceReadoutPriority>
+					<drawGUIOverlay>true</drawGUIOverlay>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RNThrown_C4CSGO_Elite"]/comps</xpath>
```

Comment:
Is this the Item or the Projectile? If it's the Item, this Explosive Comp should probably go away, since C4 is _generally_ insensitive enough not set off without a detonator or blasting cap.

Reply:
This is for the item, so I've removed the comp in 3e4cd8a